duckduckgo/iOS
# Prepare release workflow

Task/Issue URL: https://app.asana.com/0/1200194497630846/1203749968800720/f

**Description**:
The script that:
- prepares release or hotfix branch,
- commits necessary changes (app version update, build version update, embedded files update, etc.)
- creates a PR against release branch

**Steps to test this PR**:
1. Make sure prepare_release.sh, Fastlane/Fastfile and scripts/assets/appstore-release-template are untracked (so when the script changes branches they remain the same). You may need to locally commit Fastfile change on develop for the same reason.
2. Run the script without arguments, with broken arguments, with all the flags and make sure the outcome is correct.
